src/displayapp/Colors.cpp

```diff
@@ -39,7 +39,9 @@ lv_color_t Pinetime::Applications::Convert(Pinetime::Controllers::Settings::Colo
       return LV_COLOR_MAKE(0xb0, 0x0, 0xb0);
     case Pinetime::Controllers::Settings::Colors::Orange:
       return LV_COLOR_ORANGE;
+    case Pinetime::Controllers::Settings::Colors::Pink:
+      return LV_COLOR_MAKE(0xFF, 0xAE, 0xC9);
     default:
       return LV_COLOR_WHITE;
   }
-}
+}
```

Comment:
Please don't remove the last empty line at the end of the files. Try to keep the commits clean.

```suggestion
}

```